packages/pwa/app/utils/routes-utils.test.js

```diff
@@ -33,12 +43,191 @@ describe('configureRoutes', function() {
+        const paths = configuredRoutes.map((route) => route.path)
+        expect(paths).toEqual(expectedPathsResult)
     })
 })
+
+const expectedPathsResult = [
+    '/',
+    '/RefArch/en-US/category/:categoryId',
+    '/RefArch/en/category/:categoryId',
+    '/RefArch/fr-FR/category/:categoryId',
+    '/RefArch/fr/category/:categoryId',
+    '/RefArch/it-IT/category/:categoryId',
+    '/RefArch/it/category/:categoryId',
+    '/RefArch/en-HK/category/:categoryId',
+    '/RefArch/en-GB/category/:categoryId',
+    '/RefArch/uk/category/:categoryId',
+    '/us/en-US/category/:categoryId',
+    '/us/en/category/:categoryId',
+    '/us/fr-FR/category/:categoryId',
+    '/us/fr/category/:categoryId',
+    '/us/it-IT/category/:categoryId',
+    '/us/it/category/:categoryId',
+    '/us/en-HK/category/:categoryId',
+    '/us/en-GB/category/:categoryId',
+    '/us/uk/category/:categoryId',
+    '/eu/en-US/category/:categoryId',
+    '/eu/en/category/:categoryId',
+    '/eu/fr-FR/category/:categoryId',
+    '/eu/fr/category/:categoryId',
+    '/eu/it-IT/category/:categoryId',
+    '/eu/it/category/:categoryId',
+    '/eu/en-HK/category/:categoryId',
+    '/eu/en-GB/category/:categoryId',
+    '/eu/uk/category/:categoryId',
+    '/RefArchGlobal/en-US/category/:categoryId',
+    '/RefArchGlobal/en/category/:categoryId',
+    '/RefArchGlobal/fr-FR/category/:categoryId',
+    '/RefArchGlobal/fr/category/:categoryId',
+    '/RefArchGlobal/it-IT/category/:categoryId',
+    '/RefArchGlobal/it/category/:categoryId',
+    '/RefArchGlobal/en-HK/category/:categoryId',
+    '/RefArchGlobal/en-GB/category/:categoryId',
+    '/RefArchGlobal/uk/category/:categoryId',
+    '/global/en-US/category/:categoryId',
+    '/global/en/category/:categoryId',
+    '/global/fr-FR/category/:categoryId',
+    '/global/fr/category/:categoryId',
+    '/global/it-IT/category/:categoryId',
+    '/global/it/category/:categoryId',
+    '/global/en-HK/category/:categoryId',
+    '/global/en-GB/category/:categoryId',
+    '/global/uk/category/:categoryId',
+    '/RefArch/category/:categoryId',
+    '/us/category/:categoryId',
+    '/eu/category/:categoryId',
+    '/RefArchGlobal/category/:categoryId',
+    '/global/category/:categoryId',
+    '/en-US/category/:categoryId',
+    '/en/category/:categoryId',
+    '/fr-FR/category/:categoryId',
+    '/fr/category/:categoryId',
+    '/it-IT/category/:categoryId',
+    '/it/category/:categoryId',
+    '/en-HK/category/:categoryId',
+    '/en-GB/category/:categoryId',
+    '/uk/category/:categoryId',
+    '/category/:categoryId'
+]
+const mockConfig = {
+    app: {
+        defaultSite: 'RefArch',
+        url: {
+            site: 'none',
+            locale: 'none'
+        },
+        hosts: [
+            {
+                domain: 'www.domain-1.com',
+                defaultSite: 'RefArch',
+                sites: [
+                    {
+                        id: 'RefArch',
+                        alias: 'us',
```

Comment:
the alias is not required, id will be the fallback value in case there is not alias. The alias is required to be unique per site, it does not make sense to me to have multiple alias for same site, how would you build the urls to navigate across the site with multi values of aliases? 🤔